R/article.R

```diff
@@ -223,6 +223,28 @@ glossa_article <- function(..., keep_tex = TRUE, latex_engine = "xelatex") {
   format
 }
 
+#' @section `glossapx_article`: Format for creating submissions to Glossa Psycholinguistics.
+#' Submission structure from <https://escholarship.org/uc/glossapsycholinguistics/structure_of_submission>
+#'    Template is adapted from the Glossa rticles template.
+#' @export
+#' @rdname article
+glossapx_article <- function(..., keep_tex = TRUE, latex_engine = "xelatex") {
+  format <- pdf_document_format(
+    "glossa",
+    keep_tex = keep_tex, latex_engine = latex_engine, ...
+  )
```

Comment:
Shouldn't it be  `glossapx` here ? You are not using the correct template I believe here

```suggestion
  format <- pdf_document_format(
    "glossapx",
    keep_tex = keep_tex, latex_engine = latex_engine, ...
  )
```

But are the template different ?

Reply:
I would like to clarify where do you think those change go ? From what I can see in the PR, it seems nothing is different in the `template.tex` or the associated resources, and things are different only in the `skeleton.Rmd`. Is that so ? 

I understand now that there may not be an easy way to : 

* Have a `glossapsycholx_article()` function in the package
* but none of the resources needs to be copied from glossa as template is the same. 

There should be a way to reuse the `glossa` template, but provide another skeleton.  If this is not possible easily already, I need to provide a way for this to happen. 

So, probably clear question is: Is this PR aiming to offer a `skeleton.Rmd` specific to `Glossa Psycholinguistics` journal which could use `glossa_article()` really ? 

My understanding is that `Glossa Psycholinguistics` is only some configuration variations from default glossa. Is that right ?

Reply:
In the PR currently, there are 
* differences in the glossa.cls file of a few lines 
diff glossa/skeleton/glossa.cls glossapx/skeleton/glossa.cls
< \RequirePackage[font=sf,labelfont=bf,labelsep=colon,justification=raggedright,singlelinecheck=off,justification=centering]{caption}
---
> \RequirePackage[font=sf,labelfont=bf,labelsep=colon,justification=raggedright,singlelinecheck=off]{caption}
498a499,502
> 
> \renewcommand{\thesection}{\arabic{section}.}
> \renewcommand{\thesubsection}{\arabic{section}.\arabic{subsection}}
> \renewcommand{\thesubsubsection}{\arabic{section}.\arabic{subsection}.\arabic{subsubsection}}

* apa7.csl for references in the skeleton 
* differences to skeleton.Rmd

I agree that the resources/template.tex is the same, so I just need a different skeleton but it's more than just the skeleton.Rmd since it needs the different cls and csl files. 

I don't know if these would count as configuration variations?

Reply:
Thanks for the clarification. 

I also read the following page: https://escholarship.org/uc/glossapsycholinguistics/structure_of_submission
and noticed this 
> Glossa Psycholinguistics  follows many of the style guidelines specified for Glossa: a journal of general linguistics. However, due to important differences **Glossa: General  stylesheets or template files should not be used** for a Glossa Psycholinguistics submission, as detailed below.

I understand there are important differences now. I need to understand the maintainance part of this. 

> differences in the glossa.cls file of a few lines

Did you make the adaptation ? Is there a `glossapsycholx` LaTeX template we can use somewhere ? 
How to we maintain the file ? How to update ? What is the source of truth for this template ? 

Is there anything to change to follow the guideline on their website ? I don't see any mention of original `glossa.cls` in their website. 

> I agree that the resources/template.tex is the same, so I just need a different skeleton but it's more than just the skeleton.Rmd since it needs the different cls and csl files.

I don't know if these would count as configuration variations?

Different CSL is a configuration with the `csl:`  field in YAML. 

Regarding `cls`, does the line need to be inserted into the CLS file explicitly ? 
Can't it be in the `template.tex` directly ? 

If this is really the same resource, but some different skeleton.Rmd and format function, I can think of how to allow that in **rticles**


Thanks a lot for the discussion, I think this is the first case we have like this. Really interesting.